components/raftstore/src/store/fsm/store.rs

```diff
@@ -2367,6 +2367,8 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> StoreFsmDelegate<'a, EK, ER
         let snap_stats = self.ctx.snap_mgr.stats();
         stats.set_sending_snap_count(snap_stats.sending_count as u32);
         stats.set_receiving_snap_count(snap_stats.receiving_count as u32);
+        stats.set_snapshot_stats(::protobuf::RepeatedField::from_vec(snap_stats.stats));
```

Comment:
Will this pr make the msg of the heartbeat very large?

Reply:
yes, I will ignore the record that  duration of snapshot is less than 1s, so the total should be less than store heartbeat interval (default 10).